definitions/pipelines/somatic_wgs.cwl

```diff
@@ -0,0 +1,529 @@
+        secondaryFiles: [.tbi]
+    strelka_filtered_vcf:
+        type: File
+        outputSource: detect_variants/strelka_filtered_vcf
+        secondaryFiles: [.tbi]
+    varscan_unfiltered_vcf:
+        type: File
+        outputSource: detect_variants/varscan_unfiltered_vcf
+        secondaryFiles: [.tbi]
+    varscan_filtered_vcf:
+        type: File
+        outputSource: detect_variants/varscan_filtered_vcf
+        secondaryFiles: [.tbi]
+    pindel_unfiltered_vcf:
+        type: File
+        outputSource: detect_variants/pindel_unfiltered_vcf
+        secondaryFiles: [.tbi]
+    pindel_filtered_vcf:
+        type: File
+        outputSource: detect_variants/pindel_filtered_vcf
+        secondaryFiles: [.tbi]
+    docm_filtered_vcf:
+        type: File
+        outputSource: detect_variants/docm_filtered_vcf
+        secondaryFiles: [.tbi]
+    final_vcf:
+        type: File
+        outputSource: detect_variants/final_vcf
+        secondaryFiles: [.tbi]
+    final_filtered_vcf:
+        type: File
+        outputSource: detect_variants/final_filtered_vcf
+        secondaryFiles: [.tbi]
+    final_tsv:
+        type: File
+        outputSource: detect_variants/final_tsv
+    vep_summary:
+        type: File
+        outputSource: detect_variants/vep_summary
+    tumor_snv_bam_readcount_tsv:
+        type: File
+        outputSource: detect_variants/tumor_snv_bam_readcount_tsv
+    tumor_indel_bam_readcount_tsv:
+        type: File
+        outputSource: detect_variants/tumor_indel_bam_readcount_tsv
+    normal_snv_bam_readcount_tsv:
+        type: File
+        outputSource: detect_variants/normal_snv_bam_readcount_tsv
+    normal_indel_bam_readcount_tsv:
+        type: File
+        outputSource: detect_variants/normal_indel_bam_readcount_tsv
+# ##copy number calling
+#     intervals_antitarget:
+#         type: File?
+#         outputSource: cnvkit/intervals_antitarget
+#     intervals_target:
+#         type: File?
+#         outputSource: cnvkit/intervals_target
+#     normal_antitarget_coverage:
+#         type: File
+#         outputSource: cnvkit/normal_antitarget_coverage
+#     normal_target_coverage:
+#         type: File
+#         outputSource: cnvkit/normal_target_coverage
+#     reference_coverage:
+#         type: File?
+#         outputSource: cnvkit/reference_coverage
+#     cn_diagram:
+#         type: File?
+#         outputSource: cnvkit/cn_diagram
+#     cn_scatter_plot:
+#         type: File?
+#         outputSource: cnvkit/cn_scatter_plot
+#     tumor_antitarget_coverage:
+#         type: File
+#         outputSource: cnvkit/tumor_antitarget_coverage
+#     tumor_target_coverage:
+#         type: File
+#         outputSource: cnvkit/tumor_target_coverage
+#     tumor_bin_level_ratios:
+#         type: File
+#         outputSource: cnvkit/tumor_bin_level_ratios
+#     tumor_segmented_ratios:
+#         type: File
+#         outputSource: cnvkit/tumor_segmented_ratios
```

Comment:
We could probably remove these commented-out sections entirely.  They're around in the git history and other files if we need to bring them back.